enos/enos-scenario-proxy.hcl

```diff
@@ -131,7 +148,8 @@ scenario "proxy" {
     }
 
     variables {
-      ami_id          = step.ec2_info.ami_ids[matrix.arch][matrix.distro][global.distro_version[matrix.distro]]
+      ami_id = step.ec2_info.ami_ids[matrix.arch][matrix.distro][global.distro_version[matrix.distro]]
```

Comment:
Nit: This formatting seems off compared to the rest

Reply:
Ah, good catch, it was because I missed a commented out line underneath that needed to be cleaned up. Will `enos fmt` again.